app/views/pages/release-notes/2020/APRIL/13.liquid

```diff
@@ -0,0 +1,20 @@
+---
+converter: markdown
+metadata:
+  title: Improved authorization scheme, fixes
+  description: April 13, 2020
+  skip_contribute_button: true
+  last_edit: false
+slug: release-notes/13-APRIL-2020
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__improved">IMPROVED</h4>
+
+* Update error message when rendering a form with invalid configuration.
+* Improved authorization scheme (for `pos-cli`) based on OAuth tokens - generated tokens will be scoped to one Instance. The old scheme is kept for backwards compatibility.
```

Comment:
BTW This is HUGE and deserves separate article ;)

Reply:
Thanks, I'm opening a ticket.